lib/webmock/request_execution_verifier.rb

```diff
@@ -34,26 +34,26 @@ def does_not_match?
     end
 
     def description
-      "request #{request_pattern.to_s} #{quantity_phrase.strip}"
+      "request #{request_pattern} #{quantity_phrase.strip}"
     end
 
     def failure_message
-      failure_message_phrase(is_negated=false)
+      failure_message_phrase(_is_negated = false)
```

Comment:
hi @keram, thank you for pointing it out.

I wrote `_is_negated` for like that:
- keyword arguments are not using in this project
- it is better wrote `_is_negated`  is clearly

But come to think of it, `_is_negated` is strange as arguments.
I like style of keyword arguments better, but it is for ruby 2.0 later.
And webmock supports MRI 1.8.7 later.
So I will remove it from arguments.